specification/applicationinsights/resource-manager/readme.md

```diff
@@ -222,6 +222,17 @@ directive:
   
 ```
 
+### Tag: package-2021-07-only
+
+These settings apply only when `--tag=package-2021-07-only` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-07-only'
+input-file:
+  - Microsoft.Insights/stable/2021-03-08/workbooks_API.json
+  - Microsoft.Insights/stable/2021-03-08/workbookOperations_API.json
+  - Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json
```

Comment:
They are in different folders and I advise to use same folder like : `package-2021-03-only`
![image](https://user-images.githubusercontent.com/70930885/129127128-e63158a2-af45-4e53-8a37-df8adec7fdbf.png)

Reply:
added a new tag=package-preview-2021-07 and copied all the RP from package-preview-2020-06 and updated the new workbooks paths